backend/inspirehep/legacy/views.py

```diff
@@ -0,0 +1,46 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+from flask import Blueprint, abort, redirect, request
+from sqlalchemy.orm.exc import NoResultFound
+
+from inspirehep.pidstore.api import PidStoreBase
+from inspirehep.records.utils import get_pid_for_pid
+
+blueprint = Blueprint("inspirehep_legacy", __name__, url_prefix="")
+
+
+@blueprint.route("/record/<path:recid>", methods=("GET",))
+def redirect_record(recid):
+    endpoint = PidStoreBase.get_endpoint_from_recid(recid)
+    if not endpoint:
+        abort(404)
+
+    return redirect(f"/{endpoint}/{recid}", 301)
+
+
+@blueprint.route("/author/profile/<path:bai>", methods=("GET",))
+def redirect_author(bai):
+    recid = get_pid_for_pid("bai", bai, "recid")
+    if not recid:
+        abort(404)
+
+    return redirect(f"/authors/{recid}", 301)
+
+
+@blueprint.route("/search", methods=("GET",))
+def redirect_query():
+    collection_equivalence = {
+        "HEP": "literature",
+        "HepNames": "authors",
+        "Conferences": "conferences",
+        "Jobs": "jobs",
+    }
+
+    collection = request.args.get("cc")
```

Comment:
Search in literature or coming soon message won't provide them with the information they want. Since we are keeping legacy for 2 months and we are planning to implement institutions/experiments in the meantime, it'd be better to redirect them to legacy for the time being.